layers/+tools/transmission/README.org

```diff
@@ -63,52 +64,68 @@ Add =(transmission :variables transmission-auto-refresh-all t)= to
+| ~K~         | Move torrent up the queue by one.                                 |
+| ~L~         | Move torrent to the bottom of the queue.                          |
+| ~SPC m m~   | Move torrent at point or in region to a new location.             |
+| ~SPC m q~   | Quit and bury the buffer.                                         |
+| ~SPC m v~   | Verify torrent at point or in region.                             |
+| ~SPC m D~   | Prompt to remove and delete torrent at poit or torrens in region. |
+| ~SPC m R~   | Prompt to remove torrent at point or torrents in region.          |
+| ~SPC m S~   | Sort torrents according to the column at point.                   |
+| ~SPC m a a~ | Add torrent by filename, URL, magnet link, or info hash.          |
+| ~SPC m a t~ | Add announce URLs to torrent or torrents.                         |
+| ~SPC m g i~ | Open a =transmission-info-mode= buffer for torrent at point.      |
+| ~SPC m g f~ | Open a =transmission-files-mode= buffer for torrent at point.     |
+| ~SPC m g p~ | Open a =transmission-peers-mode= buffer for torrent at point.     |
+| ~SPC m s p~ | Set bandwidth priority of torrent(s) at point or in region.       |
+| ~SPC m s d~ | Set global download speed limit in kB/s.                          |
+| ~SPC m s l~ | Set label(s) for marked torrent(s) or torrent at point.           |
+| ~SPC m s r~ | Set global seed ratio limit.                                      |
+| ~SPC m s u~ | Set global upload speed limit in kB/s.                            |
+| ~SPC m t m~ | Toggle mark on torrent(s) at point or in region.                  |
+| ~SPC m t s~ | Toggle torrent between started and stopped.                       |
+| ~SPC m t t~ | Toggle =transmission-turtle-mode=.                                |
 
 ** =transmission-info-mode=
 
 | Key binding | Description                                                   |
 |-------------+---------------------------------------------------------------|
 | ~SPC m c~   | Copy magnet link of current torrent.                          |
-| ~SPC m s d~ | Set download limit of torrent(s) at point in kB/s.            |
-| ~SPC m g p~ | Open a =transmission-peers-mode= buffer for torrent at point. |
-| ~SPC m s l~ | Set seed ratio limit of torrent(s) at point.                  |
 | ~SPC m m~   | Move torrent at point or in region to a new location.         |
-| ~SPC m a a~ | Add announce URLs to torrent or torrents.                     |
 | ~SPC m T~   | Remove trackers from torrent at point by ID or announce URL.  |
-| ~SPC m s u~ | Set upload limit of torrent(s) at point in kB/s.              |
+| ~SPC m a t~ | Add announce URLs to torrent or torrents.                     |
+| ~SPC m g p~ | Open a =transmission-peers-mode= buffer for torrent at point. |
+| ~SPC m s d~ | Set download limit of torrent(s) at point in kB/s.            |
+| ~SPC m s l~ | Set seed ratio limit of torrent(s) at point.                  |
```

Comment:
we should also add `SPC m s L` for adding labels.

Reply:
Oh, it has already been added but I forgot to correct this table! What a blunder on my end!

Reply:
Corrected the README file